hacking/hacking.md

```diff
@@ -50,26 +50,19 @@ This will result in the following binary files:
 ## Alternative Hardware
 
 A number of users have contributed alternative hardware designs that are compatible with the native firmware.
-Some of these designs are in the [hardware repo](https://github.com/blackmagic-debug/blackmagic-hardware/tree/master/contrib). Check the `README.md` files for details. For instance, to compile a BMP for an ST-Link v2 to run as alternative to the ST firmware, compile:
+Some of these designs are in the [hardware repo](https://github.com/blackmagic-debug/blackmagic-hardware/tree/master/contrib). Check the `README.md` files for details. For instance, to compile a BMP for an ST-Link V2 to run as alternative to the ST firmware, compile:
```

Comment:
The lowercase v in "v2" here is intentional to be consistent with how the project refers to ST-Link hardware elsewhere (including in the firmware source). This is also how ST refers to the product in much of their literature.

Reply:
Ah ok, I had found that ST refers to it as [V2](https://www.st.com/en/development-tools/st-link-v2.html). I will update to lowercase "v" again.